tests/integration_tests/test_cma.py

```diff
@@ -184,8 +187,16 @@ def test_init(search_space: Dict[str, BaseDistribution], x0: Dict[str, Any]) ->
                 {
                     "BoundaryHandler": cma.BoundTransform,
                     "bounds": [
-                        [-0.5, -1.0, -1.5, -2.0, math.log(1.5), math.log(0.001), -2,],
-                        [1.5, 11.0, 1.5, 4.0, math.log(16.5), math.log(0.1), 2],
+                        [-0.5, -1.0, -1.5, -2.0, math.log(1.5), math.log(0.001), -2],
```

Comment:
I'm either fine, but it might be good to keep as it is.
```suggestion
                        [-0.5, -1.0, -1.5, -2.0, math.log(1.5), math.log(0.001), -2,],
```

Reply:
Thank you for the suggestion 👍  Honestly, I also either is fine but feels more natural that trimmed the comma when `list` is written in one line. So I make resolved this without applying the changes.

I guess the comma is added when `black` formatted from multiple lines into a single line like following.

```
[
    0.5,
    1.0,
]
↓ formatted
[0.5, 1.0,]
```